composer/datasets/in_context_learning_evaluation.py

```diff
@@ -917,7 +749,7 @@ class InContextLearningCodeEvalDataset(Dataset):
         supported by :meth:`composer.utils.maybe_create_object_store_from_uri`. Dataset must consist of rows of JSON data points with "task_id",
         "prompt", "entry_point", "canonical_solution", "test", "test_inputs", and "test_outputs". See tests/datasets/local_data/human_eval_small.jsonl.
         tokenizer (Union[transformers.PreTrainedTokenizer, transformers.PreTrainedTokenizerFast]): The tokenizer used to map between strings and token ids
-        batch_size (int): Size of a batch used for eval
+        ? batch_size (int): Size of a batch used for eval
```

Comment:
?

Reply:
lol all the classes claim to take in batch size but they dont, and I was going thru the doc string to see which params I had changed and couldn't find batch_size and this was my note of that

Reply:
I see your point haha